src/app/pro/billing/page.tsx

```
@@ -0,0 +1,226 @@
+          <CardTitle>Payment Methods</CardTitle>
+          <CardDescription>
+            Manage your saved payment methods
+          </CardDescription>
+        </CardHeader>
+        <CardContent>
+          <div className="text-center py-8">
+            <p className="text-muted-foreground mb-4">
+              Since Pro is a lifetime purchase, no payment method is stored.
+            </p>
+            <p className="text-sm text-muted-foreground">
+              For future purchases or upgrades, you'll be able to add payment methods here.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)